docs/source/user/conversion.rst

```diff
@@ -0,0 +1,97 @@
+.. _conversion:
+
+===============
+Data Conversion
+===============
+
+By default PyMySQL will convert MySQL data types into native Python types. The
+full list of conversions is:
+
+=============== =============================================
+MySQL type      Python type
+=============== =============================================
+``BIT``         [No conversion]
+``TINY``        ``Int``
+``SHORT``       ``Int``
+``LONG``        ``Int``
+``FLOAT``       ``Float``
+``DOUBLE``      ``Float``
+``LONGLONG``    ``Int``
+``INT24``       ``Int``
+``YEAR``        ``Int``
+``TIMESTAMP``   ``datetime.datetime`` or ``datetime.date`` *
+``DATETIME``    ``datetime.datetime`` *
+``TIME``        ``datetime.timedelta`` *
+``DATE``        ``datetime.date`` *
+``SET``         ``Set``
+``BLOB``        ``String``
+``TINY_BLOB``   ``String``
+``MEDIUM_BLOB`` ``String``
+``LONG_BLOB``   ``String``
+``STRING``      ``String``
+``VAR_STRING``  ``String``
+``VARCHAR``     ``String``
+``DECIMAL``     ``Decimal``
+``NEWDECIMAL``  ``Decimal``
+=============== =============================================
+
+Items marked * are returned as ``None`` if they are of an invalid format or are
+invalid dates. For example, these valid ``DATETIME`` values::
```

Comment:
Please add note about (sorry, my English is bad,  please rewrite to more good sentence):

* Customizing converter is **hack** and **not recommended**.  It's integrated implementation detail very tightly, so it's difficult to keep backward compatibility sometimes.
*  You should try converting at higher layer like ORM always when possible.
*  If your DB contains broken data, you should fix it, while you can use converter hack for temporal workaround.